app/uk.gov.hmrc.helptosavestridefrontend/controllers/StrideController.scala

```diff
@@ -48,7 +49,7 @@ class StrideController @Inject() (val authConnector:       AuthConnector,
                                   messageApi:              MessagesApi)(implicit val transformer: NINOLogMessageTransformer)
```

Comment:
my intellij complains about `implicit FrontendAppConfig` is not found - 
may be better move this `val frontendAppConfig:   FrontendAppConfig` to implicit declaration next to it ?